pipeline/inputs/statsd.md

```diff
@@ -16,6 +16,12 @@ The plugin supports the following configuration parameters:
 | Listen | Listener network interface. | 0.0.0.0 |
 | Port | UDP port where listening for connections | 8125 |
 | Threaded | Indicates whether to run this input in its own [thread](../../administration/multithreading.md#inputs). | `false` |
+| Metrics | Ingest as metrics type of events. | `off` |
```

Comment:
So I think this means when we ingest records they will be marked as a metrics type rather than a logs type right?

Reply:
```suggestion
| Metrics | Ingested record will be marked as a metric record rather than a log record. | `off` |
```